lib/fog/openstack/requests/compute/list_volume_attachments.rb

```diff
@@ -0,0 +1,31 @@
+module Fog
+  module Compute
+    class OpenStack
+      class Real
+        def list_volume_attachments(server_id)
+          request(
+            expects:  200,
+            method:   'GET',
+            path:     format('servers/%s/os-volume_attachments', server_id)
+          )
+        end
+      end
+
+      class Mock
+        def list_volume_attachments(server_id)
+          Excon::Response.new(
+            body: {
+              volumeAttachments: [{
```

Comment:
Given that we only support Ruby >= 1.9.x, I believe we should be using the new syntax in place of hash rockets